WebRoot/js/ajax/dwt/widgets/DwtBaseDialog.js

```diff
@@ -242,7 +242,8 @@ function(loc) {
 	kbMgr.pushTabGroup(this._tabGroup);
 	kbMgr.pushDefaultHandler(this);
 
-	DwtShell.getShell().addListener(DwtEvent.CONTROL, this._resizeHdlr.bind(this));
+	this.shellResizeHandler = this._resizeHdlr.bind(this);
```

Comment:
Maybe not 
TypeError: Cannot read property 'isListenerRegistered' of undefined at DwtShell._resizeHdlr 

if (shell.isListenerRegistered(DwtEvent.CONTROL)) {

At least its another puzzle.